src/Bicep.Core/Parsing/ParamsParser.cs

```diff
@@ -74,13 +75,42 @@ public override ProgramSyntax Program()
         private UsingDeclarationSyntax UsingDeclaration()
         {
             var keyword = ExpectKeyword(LanguageConstants.UsingKeyword);
+
+            SyntaxBase expression = reader.Peek().Type switch
+            {
+                TokenType.EndOfFile or
+                TokenType.NewLine or
+                TokenType.NoneKeyword => WithRecovery(
+                    () => new NoneLiteralSyntax(
+                        Expect(
+                            TokenType.NoneKeyword,
+                            e => e.ExpectedKeyword(LanguageConstants.NoneKeyword)
+                        )
+                    ),
+                    GetSuppressionFlag(keyword),
+                    TokenType.NewLine
+                ),
+                TokenType.StringComplete => ThrowIfSkipped(this.InterpolableString, b => b.ExpectedFilePathString()),
+                _ => Skip(reader.Read(), b => b.ExpectedSymbolListOrWildcard()),
```

Comment:
Why not just keep the existing logic for the 'fallback' case - e.g.:
```csharp
var path = reader.Peek().Type switch {
    TokenType.NoneKeyword => ...new logic...
    _ => ...old logic...
};

Reply:
I think with the last commits, we addressed this comment, too 👍